bindings/node/native/src/pre_tokenizers.rs

```diff
@@ -188,7 +188,7 @@ fn metaspace(mut cx: FunctionContext) -> JsResult<JsPreTokenizer> {
 /// split(invert: bool = false)
 fn split(mut cx: FunctionContext) -> JsResult<JsPreTokenizer> {
     let pattern: String = cx.extract::<String>(0)?;
-    let behavior: SplitDelimiterBehavior = cx.extract::<JsSDB>(1)?.0;
+    let behavior: JsSplitDelimiterBehavior = cx.extract::<JsSplitDelimiterBehavior>(1)?;
```

Comment:
Wasn't sure if that's the best way to do it with `JsSplitDelimiterBehavior` ...

Reply:
It is! I tried having `serde` use the original enum, and rename the fields to camelCase automatically but it didn't work!

Reply:
Doesn't seem to be implemented: https://github.com/serde-rs/serde/issues/1061